src/cpx_model.jl

```diff
@@ -174,6 +174,57 @@ function set_warm_start!(model::Model, indx::IVec, val::FVec, effortlevel::Integ
     end
 end
 
+function add_warm_start!(model::Model, x::Vector{Vector{Float64}}, efforts::Vector{Cint})
```

Comment:
I'm not crazy about the naming here. Probably better to write this as ``set_warm_starts!``.